script/verdaccio.js

```diff
@@ -0,0 +1,226 @@
+  process.chdir(path.join(`./${SDK_APP_NAME}`));
+  
+  // Update package.json version
+  let package = fs.readFileSync('package.json', 'utf-8');
+
+  const packageJson = JSON.parse(package);
+  const originalVersion = packageJson.version;
+  const versions = originalVersion.split('.').map(x => parseInt(x));
+  versions[2] += 1;
+  const newVersion = versions.join('.');
+
+  logger.log(`Changing version number from ${originalVersion} to ${newVersion}`);
+
+  // Update version to new version
+  let versionFile = fs.readFileSync('./src/versioning/Versioning.ts', 'utf-8');
+  fs.writeFileSync('./src/versioning/Versioning.ts', versionFile.replace('return VERSION.semverString;', `return '${newVersion};'`));
+
+  fs.writeFileSync('package.json', package.replace(originalVersion, newVersion));
+  package = fs.readFileSync('package.json', 'utf-8');
+
+  logger.log(`Temporarily removing prepublishOnly and postpublish`);
+  fs.writeFileSync('package.json', package.replace(`"postpublish": "script/postpublish",`, ""));
+  package = fs.readFileSync('package.json', 'utf-8');
+  fs.writeFileSync('package.json', package.replace(`"prepublishOnly": "script/publish",`, ""));
+  
+  logger.log('Installing dependencies...');
+  spawnOrFail('npm', ['install']);
+  logger.log('Building...');
+  spawnOrFail('npm', ['run', 'build']);
+
+  try {
+    // Cleanup package before publishing new
+    logger.log('Unpublishing any existing packages from verdaccio.');
+    spawnOrFail('npm', [`unpublish`, '--registry', `${verdaccioEndpoint}`, '--force']);
+  } catch {
+    logger.log('No existing npm module in verdaccio.');
+  }
+  logger.log('Publishing package to verdaccio.');
+  spawnOrFail('npm', [`publish`, '--registry', `${verdaccioEndpoint}`]);
+  
+  process.chdir('./demos/browser');
+
+  spawnOrFail('npm', [`uninstall ${SDK_APP_NAME}`]);
+
+  //  Install SDK components package from local repo.
+  logger.log(`Installing ${SDK_APP_NAME} from local verdaccio server...`);
+  spawnOrFail('npm', [`install ${SDK_APP_NAME} --registry ${verdaccioEndpoint}`]);
+
+  let meetingV2File = fs.readFileSync('app/meetingV2/meetingV2.ts', 'utf-8');
+  fs.writeFileSync('app/meetingV2/meetingV2.ts', meetingV2File.replace('../../../../src/index', 'amazon-chime-sdk-js'));
+
+  logger.log(`Installing...`);
+  cp.execSync('cd ../.. && npm run build');
+  cp.execSync('npm install');
+  cp.execSync('npm install aws-sdk');
+  logger.log(`Building...`);
+  cp.execSync('npm run build');
+  const demoAppClientProcess = cp.exec('node server.js');
+  applicationProcesses.push(demoAppClientProcess);
+  
+  logger.log(`
+    Starting meeting demo app. Please wait for the server to start up.
+    App will be running on: http://127.0.0.1:8080/
+
+    Once application is loaded verify that its working properly.`
+  );
+  
+  logger.warn(
+    `Have you completed the demo app manual verification steps and are ready to continue with RELEASE?`
+  );
+  shouldContinuePrompt(cleanupBeforeExit());
```

Comment:
`cleanupBeforeExit` twice? Since `shouldContinuePrompt` calls `cleanupBeforeExit` as a callback it can be either of the below two right?
 1. `shouldContinuePrompt(cleanupBeforeExit);`
2. ```shouldContinuePrompt(); cleanupBeforeExit();```

Reply:
If we receive a 'yes' and there is no callback, this will call 'exit 1.' So instead, we call this callback to cleanup before exiting

Reply:
Okok, got it, also I think it should `shouldContinuePrompt(cleanupBeforeExit);` and not `shouldContinuePrompt(cleanupBeforeExit());` does it not give out error for the later?

Reply:
yeah good catch, thanks.